lib/postgres.go

```diff
@@ -1173,6 +1188,12 @@ func (postgres *Postgres) InsertTransactionsTx(tx *pg.Tx, desoTxns []*MsgDeSoTxn
 				ReceiverPublicKey:      txMeta.ReceiverPublicKey,
 			})
 
+		} else if txn.TxnMeta.GetTxnType() == TxnTypeDAOCoinLimitOrder {
+			txMeta := txn.TxnMeta.(*DAOCoinLimitOrderMetadata)
+			_ = txMeta
+			_ = metadataDAOCoinLimitOrder
```

Comment:
```suggestion
			_, _ = txMeta, metadataDAOCoinLimitOrder
```
you can keep the changes to a single line this way